README.md

```diff
@@ -131,11 +131,13 @@ The format of the file and the current defaults:
     timezone=Etc/UTC # set to desired timezone (e.g. Europe/Ljubljana)
     locales=  # a space delimited list of locales that will be generated during install (e.g. "en_US.UTF-8 nl_NL sl_SI.UTF-8")
     system_default_locale= # the default system locale to set (using the LANG environment variable)
+    ifname=eth0
     ip_addr=dhcp
     ip_netmask=0.0.0.0
     ip_broadcast=0.0.0.0
     ip_gateway=0.0.0.0
     ip_nameservers=
+    wlandriver=
```

Comment:
Just a thought: how about changing it to `drivers_to_load` (or sth like that) and make that a space seperated list?
That way we can use that to load various drivers if needed and it is also more generic. That way we could also support [USB ethernet dongles](https://github.com/debian-pi/raspbian-ua-netinst/issues/52) or things like that.
(it may cause an issue with the selector `$ifname != eth0` though)